daras_ai_v2/language_model_openai_audio.py

```
@@ -0,0 +1,266 @@
+    if tools:
+        session_data["tools"] = [
+            tool.spec["function"] | {"type": tool.spec["type"]} for tool in tools
+        ]
+    send_recv_json(ws, {"type": "session.update", "session": session_data})
+
+    if audio_data:
+        send_json(ws, {"type": "input_audio_buffer.append", "audio": audio_data})
+        send_recv_json(ws, {"type": "input_audio_buffer.commit"})
+
+
+def stream_ws_response(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)